actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -829,6 +829,39 @@ def utf8_enforcer_tag
         '<input name="utf8" type="hidden" value="&#x2713;" />'.html_safe
       end
 
+      # Creates a text input linked to a datalist. A datalist contains a set of option tags.
+      # The datalist's bahavior is similar to a select tag. Datalists allow a selection from predefined values
```

Comment:
Hey @andyzheng547! I'm scanning through your changes and I noticed a couple of misspellings and grammar issues in the documentation. I'll comment on them inline.

bahavior -> behavior